### PR TITLE
FP filter each variant caller

### LINKED ISSUE #126: Retain variant caller information in the `set` INFO field
When running the CombineVariants command the `set` INFO field retains information about each caller that contained the variant. When we run CombineVariants a second time, the only information is that it's a `filtered` variant or a `docm` variant.  

Options:

1) Keep DoCM and Variant VCF separate (con: run VEP twice)

2) Add the original, un-annotated Variant VCF as a CWL output file and keep the combined, annotated VCF as-is

3) Run the fp_filter for each caller separately (much in the same way the docm filter is separate) and combine all 5 callers (mutect, strelka, varscan, pindel, docm) at the end just before VEP annotation

4) Use a different `set` key when running CombineVariants:
https://software.broadinstitute.org/gatk/documentation/tooldocs/current/org_broadinstitute_gatk_tools_walkers_variantutils_CombineVariants.php

### PR DESCRIPTION
**Pull request by jasonwalker80**
Run fpfilter for each caller before combining variants. Closes #126 

This will depend on a forthcoming docker-cle PR, which depends on:
https://github.com/genome/fpfilter-tool/pull/3

### CODE REVIEW

File: mutect/workflow.cwl [hunk 1]
```diff
@@ -60,10 +60,24 @@ steps:
...
+    filter:
+        run: ../fp_filter/workflow.cwl
         in:
+            reference: reference
+            cram: tumor_cram
             vcf: merge/merged_vcf
         out:
+            [filtered_vcf]
+    fp_bgzip:
+        run: bgzip.cwl
```
Comment (dufeiyu) [inline mutect/workflow.cwl]: It should be "../detect_variants/bgzip.cwl".

File: mutect/workflow.cwl [hunk 2]
```diff
@@ -60,10 +60,24 @@ steps:
...
+        in:
+            file: filter/filtered_vcf
+        out:
+            [bgzipped_file]
+    fp_index:
+        run: index.cwl
```
Comment (dufeiyu) [inline mutect/workflow.cwl]: It should be "../detect_variants/index.cwl".

File: pindel/workflow.cwl [hunk 3]
```diff
@@ -84,9 +84,23 @@ steps:
...
+    filter:
+        run: ../fp_filter/workflow.cwl
         in:
+            reference: reference
+            cram: tumor_cram
             vcf: region_filter/filtered_vcf
         out:
+            [filtered_vcf]
+    fp_bgzip:
+        run: bgzip.cwl
```
Comment (dufeiyu) [inline pindel/workflow.cwl]: same as above

File: pindel/workflow.cwl [hunk 4]
```diff
@@ -84,9 +84,23 @@ steps:
...
+        in:
+            file: filter/filtered_vcf
+        out:
+            [bgzipped_file]
+    fp_index:
+        run: index.cwl
```
Comment (dufeiyu) [inline pindel/workflow.cwl]: same as above

File: strelka/workflow.cwl [hunk 5]
```diff
@@ -63,11 +63,25 @@ steps:
...
+    filter:
+        run: ../fp_filter/workflow.cwl
         in:
+            reference: reference
+            cram: tumor_cram
             vcf: region_filter/filtered_vcf
         out:
+            [filtered_vcf]
+    fp_bgzip:
+        run: bgzip.cwl
```
Comment (dufeiyu) [inline strelka/workflow.cwl]: same as above

File: strelka/workflow.cwl [hunk 6]
```diff
@@ -63,11 +63,25 @@ steps:
...
+        in:
+            file: filter/filtered_vcf
+        out:
+            [bgzipped_file]
+    fp_index:
+        run: index.cwl
```
Comment (dufeiyu) [inline strelka/workflow.cwl]: same as above

File: varscan/workflow.cwl [hunk 7]
```diff
@@ -105,9 +105,23 @@ steps:
...
+    filter:
+        run: ../fp_filter/workflow.cwl
         in:
+            reference: reference
+            cram: tumor_cram
             vcf: merge/merged_vcf
         out:
+            [filtered_vcf]
+    fp_bgzip:
+        run: bgzip.cwl
```
Comment (dufeiyu) [inline varscan/workflow.cwl]: same as above

File: varscan/workflow.cwl [hunk 8]
```diff
@@ -105,9 +105,23 @@ steps:
...
+        in:
+            file: filter/filtered_vcf
+        out:
+            [bgzipped_file]
+    fp_index:
+        run: index.cwl
```
Comment (dufeiyu) [inline varscan/workflow.cwl]: same as above

**Comment by jasonwalker80** [comment]
This requires #130 which adds bgzip support via fpfilter.pl v0.1.0

**Pull request merged**